radix-engine-interface/src/types/node_layout.rs

```diff
@@ -123,11 +123,13 @@ impl TryFrom<u8> for EpochManagerPartitionOffset {
 
 #[repr(u8)]
 #[derive(Debug, Clone, Sbor, PartialEq, Eq, Hash, PartialOrd, Ord, FromRepr)]
-pub enum EpochManagerField {
+pub enum ConsensusManagerField {
     Config,
-    EpochManager,
+    ConsensusManager,
     CurrentValidatorSet,
     CurrentProposalStatistic,
+    CurrentTimeRoundedToMinutes,
+    CurrentTime,
```

Comment:
I wonder if we should merge `CurrentTime` into `CurrentProposalStatistic` for efficiency? It's not super important, as they're both under the same node, the JMT doesn't care too much.

Reply:
Maybe best to keep them separate for abstraction reasons 🤷